docs/source/blog/derotation.md

```diff
@@ -0,0 +1,117 @@
+:blogpost: true
+:date: July 1, 2025
+:author: Laura Porta
+:location: London, UK
+:category: Blog
+:language: English
+:image: 1
+
+# Scientific Background
```

Comment:
I think you can't have two H-1 headers in the blog. Maybe make this one H-2 and put it after the opening paragraph (and before core functionality)?

Reply:
A things are now, "Scientific background" become the title of the blogpost.

Reply:
Oh no! Ok I'll change it